### DISCUSSION TITLE
Github developer pack approved, but no email nor pack available

### DISCUSSION THREAD

**Discussion by jaikrishnatavva** [community/community#56852] [GitHub Education]
### Hi everyone,

Hi everyone!
I applied for GitHub educational verification as a student. it shows approved in the manager, but I never received an email nor the benefits, what's going on?

**Comment by patty111** [comment]
Even it showed approved in the manager, your application may still be rejected. You should receive a mail if you pass the verification.  
Maybe refresh the manager or reapply (?

**Comment by jaikrishnatavva** [comment]
![image](https://github.com/community/community/assets/132254870/aa2536cf-5bc9-4e23-b913-ffb46e591143)
![image](https://github.com/community/community/assets/132254870/ad9f4077-08e3-41d1-a668-f18c1f620a67)
i dont think I can re verify

**Reply by patty111** [reply]
https://education.github.com/pack
Are you looking for this? You can check whether you are accessible by applying copilot, gitlens+, or anything on the list by yourself. They won't send a "pack" to you

**Reply by jaikrishnatavva** [reply]
I know that they won't send a "pack" to me. But I never got an email. Is this some new sorta verification? And I am not able to access any of them.

**Comment by jaikrishnatavva** [comment]
Alright, I got the email, it required patience :)

**Reply by harshks9399** [reply]
I am facing the same issue, how long did it take the mail to come????
